package.json

```
@@ -46,20 +43,19 @@
     "electron-positioner": "^4.1.0"
   },
   "devDependencies": {
-    "@biomejs/biome": "^1.9.3",
-    "@types/jest": "^25.2.3",
-    "electron": "^32.2.0",
-    "jest": "^26.0.1",
-    "rimraf": "^3.0.2",
-    "standard-version": "^8.0.0",
-    "ts-jest": "^26.0.0",
-    "typedoc": "^0.17.7",
-    "typedoc-plugin-markdown": "^2.2.17",
-    "typedoc-plugin-no-inherit": "^1.1.10",
-    "typescript": "^4.6.2"
+    "@biomejs/biome": "^1.9.4",
+    "@types/jest": "^29.5.14",
+    "electron": "^34.3.0",
+    "jest": "^29.7.0",
+    "rimraf": "^6.0.1",
+    "standard-version": "^9.5.0",
+    "ts-jest": "^29.2.6",
+    "typedoc": "^0.27.9",
+    "typedoc-plugin-markdown": "^4.4.2",
+    "typescript": "^5.7.3"
   },
   "peerDependencies": {
-    "electron": ">=9.0.0 <33.0.0"
+    "electron": ">=9.0.0 <34.0.0"
```

```suggestion
    "electron": ">=9.0.0 <35.0.0"
```

I think without this, it won't allow e.g. 34.2.0

---

this works for me https://github.com/setchy/menubar/compare/chore/dep-updates...gobengo:menubar:chore/dep-updates